### ISSUE TITLE
Filing service error in "Filing.Replace.Bytes"

### ISSUE THREAD

**Issue by nbriggs** [devhawala/dodo#14]
Attempting to create a new file on a Dodo filing service from Interlisp (Medley) failed with "Error in Courier procedure FILING.REPLACE.BYTES (UNDEFINED.ERROR 42)"

**Comment by devhawala** [comment]
yes, the Courier-Operation Filing.ReplaceBytes ist not implemented and still on my lazy todo list.

The Pilot-based OSes (Star..GlobalView, Tajo) don't use it, so it was not so urgent, until Interlisp-D and Medley became new clients to Dodo...

so it just climbed a bit upward on the priority list

**Comment by devhawala** [comment]
just uploaded a new version of Dodo, the new `dist.zip` has a Dodo version that implements the missing procedure `ReplaceBytes`.

With this version, i was abled to open an `.tedit` file and modify it, then save the changes with _put_ to a different place on the remote file server.

**Comment by nbriggs** [comment]
Nice.  I'll probably get a chance to try it today, but if not it'll be Thursday before I can get to it.
